CardanoSharp.Wallet.Test/TransactionTests.cs

```diff
@@ -155,25 +305,173 @@ public void DeserializeMultiAssetTransaction()
         }
 
         [Fact]
-        public void SerializeTransaction()
+        public void DeserializeMultiAssetTransactionValues()
         {
-            var vectorId = "01";
-            var vectorReference = "reference.draft";
-            var outputFileName = "tx.draft";
```

Comment:
We didn't use these variables so I removed them (in the SerializeTransaction function), let me know if that's ok